contracts/OpenStream.sol

```diff
@@ -20,24 +20,40 @@ contract OpenStream is ReentrancyGuard, IOpenStream {
     uint256 public createdAt;
     ///@dev time which the payee claimed lastly
     uint256 public lastClaimedAt;
+    ///@dev Flag indicating the contract's activation status.
+    /// If `isClaimable` is `true`, the contract is considered active and ready for use.
+    /// If `isClaimable` is `false`, the contract is considered inactive and certain functions may be disabled.
```

Comment:
```suggestion
    /// If `isClaimable` is `false`, the contract is considered to be terminated and certain functions may be disabled.
```

Reply:
please use the same terminology everywhere.

We have Termination, we don't have Inactivation